Replace image processing with imagemagick to libvips (#15670)
decidim/decidim: RELEASE_NOTES.md

Reviewer: ```suggestion
sudo apt install libvips libvips-tools # or the alternative installation process for your operating system. See "3.5. Replace image processing with imagemagick to libvips"
bundle update decidim
```

You can read more about this change on PR [#15534](https://github.com/decidim/decidim/pull/15534), [#15674](https://github.com/decidim/decidim/pull/15674).

> ### 3.5. Installation of LibVips and LibVips-Tools

Reviewer: ```suggestion
### 3.5. Replace image processing with imagemagick to libvips
```

We have upgraded our image processor within the application to Libvips for speed and low memory usage.

In order to install please run the following command:

```bash
sudo apt install libvips libvips-tools
```

> You can read more about this change on PR [#15670](https://github.com/decidim/decidim/pull/15670).

Reviewer: ```suggestion
This works for Ubuntu Linux, other operating systems would need to do other command/package.

You can read more about this change on PR [#15670](https://github.com/decidim/decidim/pull/15670).
```

> 

Reviewer: ```suggestion

### 3.6. [[TITLE OF THE ACTION]]

You can read more about this change on PR [#XXXX](https://github.com/decidim/decidim/pull/XXXX).
```

You can read more about this change on PR [#15534](https://github.com/decidim/decidim/pull/15534), [#15674](https://github.com/decidim/decidim/pull/15674).

### 3.5. Replace image processing with imagemagick to libvips

> We have upgraded our image processor within the application to Libvips for speed and low memory usage.

Author: Fair and absolutely right.

In order to install please run the following command:

```bash
sudo apt install libvips libvips-tools
```

This works for Ubuntu Linux, other operating systems would need to do other command/package.

You can read more about this change on PR [#15670](https://github.com/decidim/decidim/pull/15670).

### 3.6. [[TITLE OF THE ACTION]]

You can read more about this change on PR [#XXXX](https://github.com/decidim/decidim/pull/XXXX).

> 

Author: I'm not a fan of all this description. Making implementers aware that ICO has been dropped is important and have added here: 3b5f261ca8f35921a68c0aa05a186277f06e56b9, however the rest of this information is not required and if implementers, researchers or engineers wish to know more about how Libvips works, they can conduct their own due diligence.
